packages/ckeditor5-html-support/ckeditor5-metadata.json

```diff
@@ -36,6 +36,13 @@
 			"className": "DataSchema",
 			"description": "Holds representation of the extended HTML document type definitions. Part of General HTML Support feature.",
 			"path": "src/dataschema.js"
+		},
+		{
+			"name": "HTML comment",
+			"className": "HtmlComment",
+			"description": "Preserves the HTML comments in the editor data.",
+			"docs": "features/general-html-support.html#html-comments",
+			"path": "src/htmlcomment.js"
 		}
```

Comment:
Don't we want to specify the output for this feature?

Reply:
Added.